ddtrace/contrib/_aws_lambda.py

```diff
@@ -14,7 +14,7 @@
 ~~~~~~~~~~~~~~~~~~~~
 
 This integration is configured automatically. The `datadog_lambda` package
-calls ``patch_all`` when ``DD_TRACE_ENABLED`` is set to ``true``.
+calls ``_patch_all`` when ``DD_TRACE_ENABLED`` is set to ``true``.
```

Comment:
Should we include details about patch all here? Soon this operation will be removed from our public docs